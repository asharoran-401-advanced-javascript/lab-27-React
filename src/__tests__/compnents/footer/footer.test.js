// eslint-disable-next-line strict
'use strict';

import React from 'react';
import {shallow , mount} from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../../../components/footer/footer.js';

describe(' <Footer />' , () =>{
    it('header is exist at start application' , () =>{
        let app = shallow(<Footer />);
        expect(app.find('footer').exists).toBeTruthy();
    });
    it('render correctly',()=>{
        const tree = renderer.create(<Footer />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});