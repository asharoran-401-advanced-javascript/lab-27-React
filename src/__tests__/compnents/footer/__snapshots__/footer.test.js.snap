// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` <Footer /> render correctly 1`] = `
<footer>
  Â© 2018 Code Fellows
</footer>
`;
